lib/graphQl/resolvers.js

```diff
@@ -0,0 +1,123 @@
+/* @flow weak */
+"use strict";
+var resolvers = module.exports = { };
+
+var _ = {
+  assign: require('lodash.assign')
+};
+var rerouter = require('../rerouter.js');
+var jsonApi = require('../..');
+
+
+resolvers.search = function(resourceConfig, attribute, parent, args, req, ast) {
+  var path;
+  // If we don't have a JSON:API resource, go get it
+  if (!parent) {
+    path = jsonApi._apiConfig.pathPrefix + resourceConfig.resource;
```

Comment:
I've got another PR lined up to re-lint the codebase and consider that type of thing :+1: